instance/router.go

```diff
@@ -128,7 +128,7 @@ func (in *Instance) router() *discord.MessageRouter {
 		Channel(in.ChannelID).
 		Author(DMID).
 		ContentMatchesExp(exp.search).
-		Mentions(in.Client.User.ID).
+		//Mentions(in.Client.User.ID).
```

Comment:
Looks good, if you could remove the comment as well to not clutter the code too much